SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -122,11 +122,8 @@ WorkerRequirementSet CreateReadACLForAlwaysRelevantEntities()
 	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
 
 	WorkerRequirementSet ReadACL;
-	for (const FName& WorkerType : SpatialGDKSettings->ServerWorkerTypes)
-	{
-		const WorkerAttributeSet WorkerTypeAttributeSet{ { WorkerType.ToString() } };
-		ReadACL.Add(WorkerTypeAttributeSet);
-	}
+	const WorkerAttributeSet WorkerTypeAttributeSet{ { SpatialConstants::DefaultServerWorkerType.ToString() } };
```

Comment:
Can move this line to one line above then just define the ReadACL inline?

Reply:
or use the spatial constant

Reply:
Got it!